gitpractice/javascript/task3.js

```diff
@@ -0,0 +1,13 @@
+/*http://www.codewars.com/kata/basic-mathematical-operations*/
+function basicOp(operation, value1, value2)
+    { 
+       if(operation=='+')
```

Comment:
Use switch case instead of multiple if else ladders

Reply:
okay i have used switch